misc/open_pdks/meta.yaml

```diff
@@ -0,0 +1,83 @@
+# Use `conda-build-prepare` before building for a better version string.
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('open_pdks-', '')|replace('-', '') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: open_pdks
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/RTimothyEdwards/open_pdks.git
+    git_rev: master
+  - git_url: https://github.com/google/skywater-pdk.git
+    git_rev: main    
+    folder: skywater-pdk
+  - git_url: https://github.com/google/skywater-pdk-libs-sky130_fd_sc_hd.git
+    git_rev: main
+    folder: skywater-pdk/libraries/sky130_fd_sc_hd/latest
+  - git_url: https://github.com/google/skywater-pdk-libs-sky130_fd_sc_hvl.git
+    git_rev: main    
+    folder: skywater-pdk/libraries/sky130_fd_sc_hvl/latest
+  - git_url: https://github.com/google/skywater-pdk-libs-sky130_fd_io.git
+    git_rev: main
+    folder: skywater-pdk/libraries/sky130_fd_io/latest
+  - git_url: https://github.com/google/skywater-pdk-libs-sky130_fd_pr.git
+    git_rev: main
```

Comment:
we don't actually need to do this, since `git_url` already does a recursive clone https://github.com/conda/conda-build/blob/master/conda_build/source.py#L328

Reply:
Done